src/answers-umd.js

```diff
@@ -726,6 +727,54 @@ class Answers {
       console.error(`Invalid visitor. Visitor was not set because "${JSON.stringify(visitor)}" does not have an id.`);
     }
   }
+
+  /**
+   * Initialize visibility change event listener(s) to send analytics
+   * events when user close or navigate away from a result page.
+   */
+  _initVisibilityChangeListeners (verticalKey) {
```

Comment:
Maybe this init method could go in its own file? Part of me wants to say that using a `let`-declared variable to hold state is hacky but I think I actually like it lol. Alternatively we could have a class or something

Reply:
Moved into a class!